libvast/test/bitmap_index.cpp

```diff
@@ -35,6 +36,15 @@ TEST(bool bitmap index) {
               "10010");
   CHECK_EQUAL(to_string(bmi.lookup(relational_operator::not_equal, true)),
               "01101");
+  auto builder = flatbuffers::FlatBufferBuilder{};
+  builder.Finish(pack(builder, bmi));
+  auto maybe_fb = flatbuffer<fbs::BitmapIndex>::make(builder.Release());
+  REQUIRE_NOERROR(maybe_fb);
+  auto fb = *maybe_fb;
```

Comment:
If your require an `expected` instance, we have the following pattern in unit tests:

```cpp
auto x = unbox(function_returning_expected());
```

As opposed to:
```cpp
auto maybe_x = function_returning_expected();
REQUIRE(maybe_x);
auto x = *maybe_x;
```

Reply:
I always forget about this one. Made another commit that applies this change throughout the diff.